raw_cloud_filter:
    apply_filter: true
    filter_boat_points: true
    filter_range: true
    filter_intensity: true
    debug_cloud: true

    boat_length: 3.0 # meters
    boat_width: 2.5 # meters
    max_height: 1.5 # meters
    max_xy_range: 60.0 # meters
    min_intensity: 5
pointcloud2scan:
    angle_resolution: 0.5 # degrees
